rvps/src/lib.rs

```diff
@@ -131,4 +137,19 @@ impl<T: Store> RVPSAPI for Core<T> {
             }
         }
     }
+
+    fn generate_reference_data(&self, claims: &str) -> Result<HashMap<String, Vec<String>>> {
```

Comment:
Hi, it seems that this function is to realize "giving a `<componant-name>:<evidence-digest-gathered-by-TEE>`, return the `<trusted-digests>` from RVPS". I think the logic placed here may break the modularition of RVPS. Two possible ways I think:
* Maybe this logic should be placed in AS, and something like
```rust
let mut data = HashMap::new();
for key in claims_map.keys() {
    if let Some(digest) = <rvps-obj>.get_digests(key).await? {
        data.insert(key.clone(), digest);
    } else {
        anyhow::bail!("Can not get a trust value from RVPS for {key}");
    }
    
}
```
* Change the API of RVPS `get_digests`. From `fn get_digests(&self, name: &str) -> Result<Option<TrustedDigest>>` to `fn get_digests(&self, name: &[&str]) -> Result<Vec<Option<TrustedDigest>>>;` to support request for multiple `TrustDigest`s in one request.

How do you like?